src/etc/inc/plugins.inc.d/ipsec.inc

```diff
@@ -1511,6 +1511,12 @@ function ipsec_configure_do($verbose = false, $interface = '')
                     $ealgosp1 = 'ike = ' . implode(',', array_reverse($list)) . '!';
                 }
 
+                if (!empty($ph1ent['sha256_96'])) {
+                    $sha256_96_line = "sha256_96 = {$ph1ent['sha256_96']}";
```

Comment:
@pmhausen I would fix this to "yes" to make it more explicit that this is a boolean and can't contain anything else and save the value as a boolean too (I think the other checkboxes use a similar approach).